.github/workflows/lint.yml

```diff
@@ -60,17 +60,17 @@ jobs:
           with:
             go-version: ${{env.GO_VERSION}}
             cache: true  
-        - name: Verify gomod tidy
+        - name: Verify go mod tidy
           timeout-minutes: 2
           continue-on-error: false
           run: |
+            echo "::group::go mod tidy"
             make gomodtidy
-            if ! git diff --quiet; then
-              echo "::error::Please run 'make gomodtidy' and commit the changes"
-              echo "=== files ==="
-              git diff --name-only
-              echo "=== diff ==="
-              git diff -
+            echo "::endgroup::"
```

Comment:
oh I love this !!! that is so great !!! we need to also add that to the info block, so we can Fold and Unfold each info category

Reply:
I can add an issue for using groups. Lets fix this first.